sdk/cosmos/azure-cosmos-encryption/CHANGELOG.md

```diff
@@ -1,27 +1,27 @@
 # Release History
 
-## 1.0.0-beta.5 (Unreleased)
-
+## 1.0.0-beta.5 (2021-05-20)
```

Comment:
Version headers should start with 3 hashes - to make it H2 title for Microsoft docs. 
```suggestion
### 1.0.0-beta.5 (2021-05-20)
```
Please update all of these versions to H2 title by adding 1 more # to them.

Reply:
done